packages/components/src/components/accordion/accordion-styles.ts

```diff
@@ -93,6 +93,10 @@ export const getComponentCss = (
           border: 0,
         },
       },
+      '@keyframes overflow': {
```

Comment:
Looks like this is nested within `@global`. Does it need to be there? 
I think in other cases we just have it at the bottom and not nested while referencing the animation via `$overflow`. https://cssinjs.org/jss-syntax?v=v10.10.0#keyframes-animation